Assets/MixedRealityToolkit/_Core/Managers/MixedRealityManager.cs

```diff
@@ -786,7 +848,11 @@ public void EnableManager(Type type, string managerName)
         /// <returns>An array of Managers that meet the search criteria</returns>
         public IEnumerable<IMixedRealityManager> GetManagers(Type type)
         {
-            if (type == null) { throw new ArgumentNullException(nameof(type)); }
+            if (type == null)
+            {
+                Debug.LogWarning("Unable to get managers with a type of null.");
+                return new List<IMixedRealityManager>();
```

Comment:
shouldn't this return null?

Reply:
(same question for similar returns)

Reply:
if it returns null, then code that expects to be able to enumerate over the collection, for example

foreach (IMixedRealityManager mgr in MixedRealityManager,Instance.GetManagers(type) 

will throw an exception, or have to be rewritten to cache the result of GetManagers before the loop.